hack/update-testdata.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+ROOT_DIR="$(realpath "${DIR}/..")"
+
+function update() {
+  local runtime="${1:-false}"
+  local filename="${2:-false}"
+  shift 2
+  if [ "${runtime}" != "false" ] && [ "${filename}" != "false" ]; then
+    echo "$@" >"${ROOT_DIR}/test/e2e/kwokctl/dryrun/testdata/${runtime}/${filename}"
```

Comment:
This doesn't seem to have any ability to update testdata

Reply:
Actually it does update testdata I have tested it. When we provide runtime,filename and data then it can update testdata.For example if we run the command `./hack/update-testdata.sh binary create_cluster.txt hello world` then the "hello world" gets written to the create_cluster.txt file

Reply:
Ah, if I change the behavior of kwokctl, then what do I have to do to update the testdata?

Reply:
@wzshiming we can execute this command `./hack/update-testdata.sh [runtime] [filename] [data]` wherein filename is the name of the file('create_cluster.txt','create_cluster_with_extra.txt' or 'create_cluster_with_verbosity.txt') and in data we can provide the data that will be written in that file i.e we can provide it the output of the changed behaviour and then it will be written to that file.

Reply:
I think running this should update all testdata and should have no parameters

Reply:
@wzshiming I have changed the script now please take a look